sources/core/Stride.Core.Mathematics/Vector2.cs

```diff
@@ -210,19 +210,34 @@ public float[] ToArray()
         /// <summary>
         /// Moves the first vector2 to the second one in a straight line.
         /// </summary>
-        /// <param name="First">The first point.</param>
-        /// <param name="Second">The second point.</param>
-        /// <param name="MaxDistancePerCall">The rate at which the first point is going to move towards the second point.</param>
-        public static Vector2 Goto(Vector2 First, Vector2 Second, float MaxDistancePerCall)
```

Comment:
In general, i think `MoveTo` is a better name. `Goto` might be confused with the `goto` language statement.

Reply:
alright 👍